Source/ImPlotWidgets/Public/LineFilledGraphWidget.h

```diff
@@ -16,7 +16,8 @@ class IMPLOTWIDGETS_API SFilledGraphWidget : public SCompoundWidget
 public:
 	SLATE_BEGIN_ARGS(SFilledGraphWidget)
 	{}
-	SLATE_ARGUMENT(TArray<FDataPoint>, InDataArray2D)
+	SLATE_ARGUMENT(TArray<TArray<FDataPoint>>, DataArrays)
+	SLATE_ARGUMENT(TArray<char*>, DataSetNames)
```

Comment:
I would be tempted to pass in the SetNames as FStrings and then we extract the char* from them internally - just because this is how most people will expect it to be used. FStrings use wide chars not chars, I think the TCHAR_TO_ANSI macro changes between the two. You can use the FString operator* to get the underling wide char array

Reply:
Well we probably should take them as FTexts, cause they _are_ user facing text. FText::ToString does return a reference so *MyText.ToString() won't be overly costly.

Reply:
Adding this to a new task including swapping to using a data struct